flowredux/src/commonTest/kotlin/com/freeletics/flowredux/dsl/StartStateMachineOnActionInStateTest.kt

```diff
@@ -303,6 +303,75 @@ class StartStateMachineOnActionInStateTest {
         }
     }
 
+    @Test
+    fun `actions are only dispatched to sub statemachine if they are mapped`() = suspendTest {
```

Comment:
nitpicking: maybe rename it something a bit more expressive. maybe something like 

`actions are only dispatched to sub statemachine if they are not mapped to null`()  (I think this "null" part would make it a bit more explicit for me